### ISSUE TITLE
Add capability to inject polarized and extended sources

### ISSUE THREAD

**Issue by israelmcmc** [cositools/cosipy#219]

**Comment by israelmcmc** [comment]
Status update: @krishnatejavedula started working on this. @eneights added polarization to the response module. PointSourceResponse.get_expectation(self) now has a polarization  flag. The SourceInjector can take the same flag and pass it on to the PSR.

**Comment by GallegoSav** [comment]
Would be nice if the extended source injector has a PsiChiplot option, similar to the normal source injector.

**Comment by israelmcmc** [comment]
I agree, @GallegoSav! But in addition, it would be nice if we consolidate and generalize our plotting routines. Here, for example, we could have a "plotting" module that has a function that takes (from #369) a CDSBinnedData object from any BinnedExpectationInterface (of which the polarized and unporized response are implementations) and displays the PsiChi plot.

**Comment by israelmcmc** [comment]
Closed by #459
